user-info-api/src/main/java/de/intension/api/json/HeimatOrganisation.java

```diff
@@ -0,0 +1,30 @@
+package de.intension.api.json;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import de.intension.api.UserInfoAttributeNames;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class HeimatOrganisation
+{
+
+    @JsonProperty(UserInfoAttributeNames.ID)
+    private String id;
+    @JsonProperty(UserInfoAttributeNames.NAME)
+    private String name;
+    @JsonProperty(UserInfoAttributeNames.BUNDESLAND)
+    private String bundesland;
```

Comment:
Aren't all these attributes already defined by the field name of the DTO?

Reply:
Yes, but the single source of truth is "UserInfoAttributeNames.class" and changes should be automatically reflected in the generated JSON file (just to be sure)